src/app/scripts/cac/control/cac-control-itinerary-list.js

```diff
@@ -106,10 +110,30 @@ CAC.Control.ItineraryList = (function ($, Handlebars, Utils) {
         events.trigger(eventNames.itineraryClicked, itinerary);
     }
 
+    /**
+     * Handle hover event on itinerary list item
+     */
+    function onItineraryHover() {
+        var itineraryId = this.getAttribute('data-itinerary');
```

Comment:
Borrowed that bit from the click handler.  Looking at the jQuery docs, it wasn't clear to me whether `data` was appropriate for fetching the value, or just setting it.